diff --git a/node_modules/react-native/scripts/react_native_pods.rb b/node_modules/react-native/scripts/react_native_pods.rb
index 49667ff..f526039 100644
--- a/node_modules/react-native/scripts/react_native_pods.rb
+++ b/node_modules/react-native/scripts/react_native_pods.rb
@@ -52,7 +52,7 @@ end
 def prepare_react_native_project!
   # Temporary solution to suppress duplicated GUID error.
   # Can be removed once we move to generate files outside pod install.
-  install! 'cocoapods', :deterministic_uuids => false
+  # install! 'cocoapods', :deterministic_uuids => false
 
   ReactNativePodsUtils.create_xcode_env_if_missing
 end
